plugins/msr/fu-plugin-msr.c

```diff
@@ -357,6 +359,7 @@ fu_plugin_add_security_attr_amd_sme_enabled(FuPlugin *plugin, FuSecurityAttrs *a
 
 	if (!priv->amd64_syscfg.fields.sme_is_enabled) {
 		fwupd_security_attr_set_result(attr, FWUPD_SECURITY_ATTR_RESULT_NOT_ENCRYPTED);
+		fwupd_security_attr_add_flag(attr, FWUPD_SECURITY_ATTR_FLAG_ACTION_CONFIG_FW);
```

Comment:
Should be contact vendor - they decide whether or not to offer SME.  TSME is changable in firmware

Reply:
So we have CONFIG_FW and OEM_FW_UPDATE?

Reply:
Well on second thought this is a little more complex.  SME needs OEM to set it in BIOS, but then also kernel needs to be configured to enable it.
It's more like "Fix OS" or "Contact OEM", but it's situational which one is the reason.

Reply:
And this doesn't matter if TSME is enabled or available - you can just turn that on in in firmware steup yourself..

Reply:
So potentially all three actions then?

Reply:
Well tsme is a separate attribute. So at least for this one it's only 2, but tsme it would be one.

Reply:
Gotcha, fixed.